app/controllers/grade_entry_forms_controller.rb

```diff
@@ -253,27 +252,69 @@ def student_interface
   def get_mark_columns
     grade_entry_form = GradeEntryForm.find(params[:id])
     grade_entry_items_columns = grade_entry_form.grade_entry_items
-
     c = grade_entry_items_columns.map do |column|
       {
-        id: column.name,
+        id: column.id,
         content: column.name + ' (' + column.out_of.to_s + ')',
-        sortable: true,
-        searcheable: true
       }
     end
     if grade_entry_form.show_total
       c <<
-      {
-        id: 'total_marks',
-        content: t('grade_entry_forms.grades.total') \
-                 + ' ' + grade_entry_form.out_of_total.to_s,
-      }
+        {
+          id: 'total_marks',
+          content: t('grade_entry_forms.grades.total') \
+                   + ' ' + grade_entry_form.out_of_total.to_s,
```

Comment:
Can you change this to put the total in parentheses as well, just like the other columns?